src/commands/object/generate.ts

```diff
@@ -53,14 +53,20 @@ function generate(params:GenerateOptions, session: graphene.Session){
             console.log('Invalid algorithm');
             return;
         }
-        var key = gen[alg][curve](session,params.token);
+        var key = gen[alg][curve](session, alg.toUpperCase()+'-'+curve, params.token);
```

Comment:
Move `alg.toUpperCase()+'-'+curve` to it's own line and assign it to an appropriately name variable.
The gen function call can then be more easily read.